opendocdb-cts/main.go

```diff
@@ -58,6 +64,44 @@ func fmtCommand() error {
 	return data.SaveTestSuites(tss, cli.Dir, nil)
 }
 
+// runCommand implements the "convert" command.
+func convertCommand() error {
+	fxs, err := data.LoadFixtures(cli.Dir)
+	if err != nil {
+		return err
+	}
+
+	for name, fx := range fxs {
+		err = os.MkdirAll(cli.Convert.OutDir, 0o766)
+		if err != nil {
+			return err
+		}
+
+		var f *os.File
+		f, err = os.Create(filepath.Join(cli.Convert.OutDir, name+".js"))
+		if err != nil {
+			return err
+		}
+
+		var res string
+		res, err = mongosh.ConvertFixtures(map[string]data.Fixture{name: fx})
+		if err != nil {
+			_ = f.Close()
+			return err
+		}
+
+		_, err = f.WriteString(res)
+		if err != nil {
+			_ = f.Close()
+			return err
+		}
+
+		_ = f.Close()
```

Comment:
Extracting that into function (or just closure) would simplify error handling with `defer`